src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1483,6 +1483,24 @@ public function isMethodSafe()
     }
 
     /**
+     * Checks whether the method is idempotent or not.
```

Comment:
I would go for "Checks whether or not the method is idempotent." or "Checks if the method is idempotent."